src/components/ContentEditable.native.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useRef } from 'react'
+import { ViewStyle } from 'react-native'
+import { RichEditor, RichEditorProps, RichToolbar } from 'react-native-pell-rich-editor'
+import { commonStyles } from '../style/commonStyles'
+
+interface ContentEditableProps extends RichEditorProps {
+  style: ViewStyle
+  html: string
+  disabled?: boolean
+  innerRef?: React.RefObject<HTMLDivElement>
+  isEditing?: boolean
+  forceUpdate: boolean
+  onChange: (e: string) => void
+}
+
+/**
+ * Content Editable Component.
+ */
+const ContentEditable = ({ style, html, disabled, innerRef, forceUpdate, ...props }: ContentEditableProps) => {
+  const allowInnerHTMLChange = useRef<boolean>(true)
+  const contentRef = useRef<RichEditor>(null)
```

Comment:
is this used since we ignore the toolbar?

Reply:
not necessarily, this ref is to tell the toolbar which component is the editor component. To ignore the toolbar I had to set its height to 0